lib/modules/datasource/go/index.ts

```diff
@@ -73,3 +76,14 @@ export class GoDatasource extends Datasource {
     }
   }
 }
+
+// istanbul ignore if
+if (is.string(process.env.GOPROXY)) {
+  const uri = parseUrl(process.env.GOPROXY);
+  if (uri.username) {
+    addSecretForSanitizing(uri.username, 'global');
+  }
+  if (uri.password) {
+    addSecretForSanitizing(uri.password, 'global');
+  }
+}
```

Comment:
Shouldn't we do this in `config/parse/env`? 🤔

Reply:
We could.. I was hoping to keep datasource-specific stuff out of `workers/` though. Also we don't really parse it back into config like the rest of config/parse/env.